lib/file-template.js

```diff
@@ -1,12 +1,13 @@
 'use strict'
 
-const fileTemplate=(language, version) => {
-	var config = `language: ${language}\n${language}:\n`
-	for (let i = 0; i < version.length; i++) {
-		config += `  - "${version[i]}"\n`
-	}
+const fileTemplate = (language, ...version) => {
```

Comment:
You are right, what I did was a bit generalized, but since we know the number of options and I agree, declaring it as an array makes things simpler :+1: . Will change that and commit again :). Thanks again for your thoughts :)